tests/Unit/Domain/CoordinateMaps/TestMapHelpers.hpp

```diff
@@ -260,33 +260,39 @@ void test_map3d(const Map& map) {
   const double zeta = real_dis(gen);
   CAPTURE_PRECISE(zeta);
 
-  const std::array<std::array<double, 3>, 13> test_points{{{{0.0, 0.0, 0.0}},
-                                                          {{-1.0, -1.0, -1.0}},
-                                                          {{1.0, -1.0, -1.0}},
-                                                          {{-1.0, 1.0, -1.0}},
-                                                          {{1.0, 1.0, -1.0}},
-                                                          {{-1.0, -1.0, 1.0}},
-                                                          {{1.0, -1.0, 1.0}},
-                                                          {{-1.0, 1.0, 1.0}},
-                                                          {{1.0, 1.0, 1.0}},
-                                                          {{-0.1, 0.3, 0.1}},
-                                                          {{0.5, 0.7, -0.5}},
-                                                          {{0.9, -1.0, 0.4}},
-                                                          {{xi, eta, zeta}}}};
+  const std::array<double, 3> origin3D{{0.0, 0.0, 0.0}};
+  const std::array<double, 3> random_point3D{{xi, eta, zeta}};
+  std::array<double, Map::dim> origin;
+  std::array<double, Map::dim> random_point;
+  for (size_t i = 0; i < Map::dim; i++) {
```

Comment:
get rid of xi, eta, zeta, origin3D and random_point3D and just inline (capture the seed instead of the values)